src/main/java/com/test/SwaggerConfig.java

```diff
@@ -1,73 +0,0 @@
-/*
- * package com.test;
- * 
- * import java.time.LocalDate; import java.time.LocalDateTime;
- * 
- * import
- * org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
- * import org.springframework.boot.context.properties.ConfigurationProperties;
- * import org.springframework.context.annotation.Bean; import
- * org.springframework.context.annotation.Configuration;
- * 
- * import springfox.documentation.builders.ApiInfoBuilder; import
- * springfox.documentation.builders.PathSelectors; import
- * springfox.documentation.builders.RequestHandlerSelectors; import
- * springfox.documentation.service.ApiInfo; import
- * springfox.documentation.service.Contact; import
```

Comment:
@Rinshakp 

Why is this whole file commented?
Because of this the swagger-ui.html is not accessible.
Have you verified the working functionalities of the Rest APIs through swagger?